style.css

```diff
@@ -1,20 +1,47 @@
-.outline {
-  outline: 1px solid black;
+body {
+  font-family: "Playfair Display", serif;
+  font-size: 28px;
+  background: #0000ff2b;
+  text-align: center;
 }
-.grid-container {
+
+.button__submit {
+  padding: 10px;
+  border-radius: 8px;
+  margin-bottom: 30px;
+  background: #3f51b563;
+  border-style: none;
+}
+.button__radio {
+  margin-left: 20px;
+}
+
+.section-grid {
   display: flex;
-  flex-direction: row;
-  flex-wrap: wrap;
-  width: 1600px;
-  height: 1600px;
-  outline: 1px solid black;
+  width: 800px;
```

Comment:
I'd try to make the page to always fit to the size of the display port (browser window). In the current implementation I have to scroll down to see the whole canvas.